app/src/main/res/layout/lists_with_add_content.xml

```diff
@@ -78,10 +79,12 @@
             style="@style/num_names"
             />
 
-        <ListView
+        <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/content_list"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
+            android:clipToPadding="false"
```

Comment:
We don't need clipToPadding="false", since there's no FloatingActionButton at the bottom.